python-nectar.spec

```diff
@@ -26,24 +27,43 @@ such as the default "threaded" downloader, which uses the "requests" library
 with multiple threads. Other experimental downloaders have used tools like
 pycurl and eventlets.
 
+%package -n python2-nectar
+Summary:        %{sum}
+Requires:       python-isodate >= 0.4.9
+Requires:       python-requests >= 2.4.3
+%description -n python2-nectar
+
+%package -n python3-nectar
+Summary:        %{sum}
+Requires:       python3-isodate >= 0.4.9
+Requires:       python3-requests >= 2.4.3
+%description -n python3-nectar
+
 %prep
 %setup -q
 
 %build
-%{__python} setup.py build
+%py2_build
+%py3_build
 
 %install
 rm -rf $RPM_BUILD_ROOT
 %{__python} setup.py install -O1 --skip-build --root %{buildroot}
+%{__python3} setup.py install -O1 --skip-build --root %{buildroot}
 
 %clean
 rm -rf $RPM_BUILD_ROOT
 
-%files
+%files -n python2-nectar
 %{python_sitelib}/nectar/
 %{python_sitelib}/nectar*.egg-info
 %doc COPYRIGHT LICENSE.txt README.rst
 
+%files -n python3-nectar
+%{python3_sitelib}/nectar/
+%{python3_sitelib}/nectar*.egg-info
+%doc COPYRIGHT LICENSE.txt README.rst
```

Comment:
License files should have own definition with specific macro:

%license LICENSE